### PR TITLE
fix session occurrence api

### PR DESCRIPTION
**Pull request by belwalshubham**

### CODE REVIEW

File: priv/static/swagger.json [hunk 1]
```diff
@@ -1688,7 +1688,7 @@
         ]
       }
     },
-    "/api/session-occurence": {
+    "/api/session-occurrence": {
```
Comment (Bahugunajii) [inline priv/static/swagger.json:1691]: why are we making changes in swagger.json file. It's randomly generated no?

**Review by Bahugunajii — CHANGES_REQUESTED** [review]

File: lib/dbservice_web/controllers/session_occurence_controller.ex [hunk 2]
```diff
@@ -104,10 +103,10 @@ defmodule DbserviceWeb.SessionOccurenceController do
   end
 
   swagger_path :delete do
-    PhoenixSwagger.Path.delete("/api/session-occurence/{sessionOccurenceId}")
+    PhoenixSwagger.Path.delete("/api/session-occurrence/{sessionOccurrenceId}")
 
     parameters do
-      sessionOccurenceId(:path, :integer, "The id of the session occurence record", required: true)
+      sessionOccurrenceId(:path, :integer, "The id of the session occurence", required: true)
```
Comment (Bahugunajii) [inline lib/dbservice_web/controllers/session_occurence_controller.ex:109]: why you removed record from here?

**Review by Bahugunajii — CHANGES_REQUESTED** [review]

File: priv/static/swagger.json (hunk 1 above)
Comment (belwalshubham) [reply priv/static/swagger.json]: I have reverted those changes. Please take a look.

File: priv/static/swagger.json [hunk 3]
```diff
@@ -1734,15 +1734,15 @@
         ]
       }
     },
-    "/api/session-occurence/{sessionOccurenceId}": {
+    "/api/session-occurrence/{sessionOccurrenceId}": {
```
[the remark this answers is not in the document]
Comment (belwalshubham) [reply priv/static/swagger.json]: done

**Review by Bahugunajii — APPROVED** [review]

**Pull request merged**